bin/govuk-docker

```diff
@@ -1,23 +1,44 @@
-#!/usr/bin/env ruby
+#!/bin/bash
 
-require "pathname"
-ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../Gemfile",
-                                           Pathname.new(__FILE__).realpath)
+case $1 in
+  doctor)
+    echo '*** DEPRECATED: use bundle exec bin/doctor instead ***'
+    echo 'https://github.com/alphagov/govuk-docker/blob/e00ab054d8d94132f1539b0027eefbe44e168f6f/docs/installation.md#how-to-troubleshoot-your-installation'
+    exit 1
+    ;;
 
-bundle_binstub = File.expand_path("govuk_docker_bundle", __dir__)
+  setup)
+    echo '*** DEPRECATED: use bundle exec bin/setup instead ***'
+    echo 'https://github.com/alphagov/govuk-docker/blob/e00ab054d8d94132f1539b0027eefbe44e168f6f/docs/installation.md#setup'
+    exit 1
+    ;;
 
-if File.file?(bundle_binstub)
-  if File.read(bundle_binstub, 300) =~ /This file was generated by Bundler/
-    load(bundle_binstub)
-  else
-    abort("Your `bin/bundle` was not generated by Bundler, so this binstub cannot run.
-Replace `bin/bundle` by running `bundle binstubs bundler --force`, then run this command again.")
-  end
-end
+  build)
+    echo '*** DEPRECATED: use make <service> instead ***'
+    echo 'https://github.com/alphagov/govuk-docker/blob/e00ab054d8d94132f1539b0027eefbe44e168f6f/README.md#usage'
+    exit 1
+    ;;
+esac
 
-require "rubygems"
-require "bundler/setup"
+echo '*** DEPRECATED: use exe/govuk-docker instead ***'
+echo 'https://github.com/alphagov/govuk-docker/blob/e00ab054d8d94132f1539b0027eefbe44e168f6f/docs/installation.md#setup'
+echo
```

Comment:
Should this also be in the case statement for an empty option (I say this with virtually no knowledge of bash case statements) to prevent you receiving two outputs if you have an argument.

Reply:
Ah, so this is deliberate, as I think we should print out the generic bit about using `exe/govuk-docker` before talking about any specific sub-commands that have changed. In the first case, we're pointing to the new installation docs; in the specific cases, we're pointing to the alt thing to run.